mappings/net/minecraft/entity/attribute/ClampedEntityAttribute.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_1329 net/minecraft/entity/attribute/ClampedEntityAttribute
+	COMMENT Represents an attribute with minimum and maximum value limits.
```

Comment:
```suggestion
	COMMENT Represents a type of attribute with minimum and maximum value limits.
```
Should have fixed that earlier. `Attribute` apparently are types than instances.